_tests/browser_tests/spec_suites/shared/shared_the-bureau.js

```diff
@@ -8,12 +10,16 @@ describe( 'Beta The Bureau Page', function() {
     page.get();
   } );
 
-  it( 'should properly load in a browser', function() {
+  it( 'should properly load in a browser',
+  function() {
```

Comment:
Minor: `it( … )` parameters could be aligned like:

```
it( 'should properly load in a browser',
    function() {
      expect( page.pageTitle() ).toBe( 'The Bureau' );
    } 
);
```

But maybe that makes it kinda weird with the last line of the function.